libs/ktor2client-onbehalfof/src/test/kotlin/DownstreamResourceClientTest.kt

```diff
@@ -12,4 +24,28 @@ internal class DownstreamResourceClientTest {
         assertTrue(contentTypeErLik(contenttypeMedCharset, ContentType.Application.Json))
         assertTrue(contentTypeErLik(ContentType.Application.Json, contenttypeMedCharset))
     }
+
+    @Test
+    fun `bruker client credentials viss JWT-claims sub og oid er like`() {
+        every { runBlocking { azureAdClient.getAccessTokenForResource(any()) } } returns Ok(mockk())
```

Comment:
Har vi nokre betre mønster for dette? Kjens unødig med så mykje coroutine-kode blanda inn i testen